civihr_employee_portal/views/templates/views-view-table--Documents--block.tpl.php

```diff
@@ -96,66 +96,79 @@
-            <tr <?php if ($row_classes[$row_count] || $class) {
-                print 'class="' . implode(' ', $row_classes[$row_count]) . ' ' . $class . '"';
-              } ?>>
-                <?php
-                foreach ($row as $field => $content) {
-                  $class = '';
-                  $attribute = '';
-                  if (!empty($field_classes[$field][$row_count])) {
-                    $class = sprintf('class = "%s"', $field_classes[$field][$row_count]);
-                  }
-                  if (!empty($field_attributes[$field][$row_count])) {
-                    $attribute = drupal_attributes($field_attributes[$field][$row_count]);
-                  }
-
-                  printf('<td %s %s>%s</td>', $class, $attribute, $content);
-                } ?>
-              <td data-ct-spinner data-ct-spinner-id="document-<?php print $rowID; ?>">
-                <?php if ($row['status_id'] === 'awaiting upload'): ?>
-                  <button
-                    ng-show='!document.loadingModalData'
-                    ng-click="document.modalDocument('<?php print $rowID ?>', 'staff', 'edit')"
-                    class="btn btn-sm btn-default">
-                    <i class="fa fa-upload"></i>
-                    Upload
-                  </button>
-                <?php else: ?>
-                  <button
-                    ng-show='!document.loadingModalData'
-                    ng-click="document.modalDocument('<?php print $rowID ?>', 'staff', 'view')"
-                    class="btn btn-sm btn-default">
-                    <i class="fa fa-eye"></i>
-                    View
-                  </button>
-                  <?php $showDownload = CRM_Utils_Array::value($rowID, $documentFileCount, 0) > 0 ? 'true' : 'false'; ?>
-                  <a class="btn btn-sm btn-default"
-                    <?php printf("ng-show='!document.loadingModalData && %s'", $showDownload); ?>
-                    target="_blank"
-                    ng-href="/civicrm/tasksassignments/file/zip?entityID=<?php print $rowID; ?>&entityTable=civicrm_activity"
-                  >
-                    <i class="fa fa-download"></i>
-                    Download
-                  </a>
-                <?php endif; ?>
-              </td>
-            </tr>
-          <?php endforeach; ?>
+        <?php foreach ($rows as $row_count => $row):
+          $mode = 'view';
+          $faIcon = 'fa-eye';
+          $label = 'View';
+          $rowID = strip_tags(CRM_Utils_Array::value('id', $row));
+          $status = CRM_Utils_Array::value('status_id', $row);
+
+          if($status === 'awaiting upload') {
+            $mode = 'edit';
+            $label = 'Upload';
+            $faIcon = 'fa-upload';
+          }
+
+          ?>
+          <!-- no results row -->
+          <tr id = "no-results-row" style="display:none;">
+            <td colspan="100%">There are no documents to display.</td>
+          </tr>
```

Comment:
If this is inside the rows loop, how it will be displayed when there are no documents? Won't the $rows array be empty in that case?